core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -226,7 +226,7 @@ trait WhiskActionsApi
                         val action = act.resolve(user.namespace)
                         onComplete(entitleReferencedEntities(user, Privilege.ACTIVATE, Some(action.exec))) {
                             case Success(true) =>
-                                transid.started(this, if (blocking) LoggingMarkers.CONTROLLER_ACTIVATION_BLOCKING else LoggingMarkers.CONTROLLER_ACTIVATION)
+                                transid.started(this, if (blocking) LoggingMarkers.CONTROLLER_ACTIVATION_BLOCKING else LoggingMarkers.CONTROLLER_ACTIVATION, s"${action.name}")
```

Comment:
suggestion instead of using string interpolation here: `action.name.toString`.

the action name is available in other log messages for this transid - do we need to log the action name here explicitly?